modules/azure_k8s_base/azure-k8s-base.json

```diff
@@ -23,6 +23,21 @@
       "description": "Additional configuration for nginx ingress. [Available options](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/configmap/#configuration-options)",
       "default": {}
     },
+    "cert_manager_values": {
+      "type": "object",
+      "description": "Certificate Manager helm chart additional values. [Available options](https://artifacthub.io/packages/helm/cert-manager/cert-manager?modal=values)",
+      "default": {}
+    },
+    "linkerd_values": {
+      "type": "object",
+      "description": "Linkerd helm chart additional values. [Available options](https://artifacthub.io/packages/helm/linkerd2/linkerd2/2.10.2?modal=values)",
+      "default": {}
+    },
+    "ingress_nginx_values": {
+      "type": "object",
+      "description": "Ingress Nginx helm chart additional values. [Available options](https://artifacthub.io/packages/helm/ingress-nginx/ingress-nginx/4.0.17?modal=values)",
```

Comment:
similar comment about defining a version constant to avoid hard coded the version in the url. 
not blocking.

Reply:
We want to test these things before giving these out to our users.